client/landing/gutenboarding/path.ts

```diff
@@ -122,7 +122,6 @@ export function useOnboardingFlow(): string {
 	}
 	return FLOW_ID;
 }
-
```

Comment:
Accidental?

Reply:
Hmm, I think so... VS Code UI makes it look like there's an extra space there with their annotations. Anyway, added it back and also removed the `ANCHOR_FM_FLOW_ID` constant as promised!